app/pb_kits/playbook/pb_circle_chart/_circle_chart.jsx

```diff
@@ -0,0 +1,144 @@
+  data?: Object,
+  dataLabelHtml: string,
+  dataLabels: boolean,
+  headerFormat: string,
+  id?: string,
+  innerSize: "sm" | "md" | "lg" | "none",
+  legend: boolean,
+  maxPointSize: number,
+  minPointSize: number,
+  rounded: boolean,
+  startAngle: number,
+  style: string,
+  title: string,
+  tooltipHtml: string,
+  useHtml: boolean,
+  zMin: number,
+}
+
+const CircleChart = (props: CircleChartProps) => {
+  const {
+    aria = {},
+    chartData = [{}],
+    children,
+    className,
+    colors = [],
+    data = {},
+    dataLabelHtml = '<div>{point.name}</div>',
+    dataLabels = false,
+    headerFormat = null,
+    id,
+    innerSize = 'md',
+    legend = false,
+    maxPointSize = null,
+    minPointSize = null,
+    rounded = false,
+    startAngle = null,
+    style = 'pie',
+    title = '',
+    tooltipHtml = '<span style="font-weight: bold; color:{point.color};">●</span>{point.name}: ' +
+      '<b>{point.y}</b>',
+    useHtml = false,
+    zMin = null,
+  } = props
+
+  const ariaProps = buildAriaProps(aria)
+  const dataProps = buildDataProps(data)
+
+  const innerSizeFormat = (size) => {
+    if (size == 'sm') {
+      return '35%'
+    }
+    if (size == 'md') {
+      return '50%'
+    }
+    if (size == 'lg') {
+      return '85%'
+    }
+    if (size == 'none') {
+      return '0%'
+    }
+  }
+
+  const roundedBorderWidth = rounded ? 20 : null
+  const roundedBorderColor = rounded ? null : ''
+
+  // Runs first time component Renders
+  useEffect(() => {
+    const formattedChartData = chartData.map((obj) => {
+      obj.y = obj.value
+      delete obj.value
+      return obj
+    })
+
+    new pbChart('.selector', {
+      id: id,
+      colors: colors,
+      borderColor: roundedBorderColor,
+      borderWidth: roundedBorderWidth,
+      chartData: formattedChartData,
+      title: title,
+      type: style,
+      showInLegend: legend,
+      dataLabelHtml: dataLabelHtml,
+      dataLabels: dataLabels,
+      headerFormat: headerFormat,
+      tooltipHtml: tooltipHtml,
+      useHTML: useHtml,
+      minPointSize: minPointSize,
+      maxPointSize: maxPointSize,
+      innerSize: innerSizeFormat(innerSize),
+      zMin: zMin,
+      startAngle: startAngle,
+    })
+  }, [])
+
+  const componentDidMount = useRef(false)
+
+  // Doesn't run the first time but runs every subsequent render
+  useEffect(() => {
+    if (componentDidMount.current) {
+      Highcharts.charts.forEach((chart) => {
+        if (chart.renderTo.id === id) {
+          const updatedValueArray = chartData.map((obj) => {
+            return obj.value
+          })
+          chart.series[0].setData(updatedValueArray)
+        }
+      })
+    } else {
+      componentDidMount.current = true
+    }
+  }, [chartData])
+  return (
+    <div id={`wrapper-circle-chart-${id}`}>
+      <div
+          id={id}
+          {...ariaProps}
+          {...dataProps}
+          className={classnames('pb_circle_chart', globalProps(props), className)}
```

Comment:
```suggestion
          id={id}
          className={classnames('pb_circle_chart', globalProps(props), className)}
          {...ariaProps}
          {...dataProps}
```

Reply:
Linter should have caught this one.

Reply:
This one is actually fine because it's inside of the classnames function. The rails kit is already targeting pb_circle_chart  too, so my thought is... The less I have to change, the less likely it will be to break something else. However, it is only being used in one spot on Nitro. I'm happy to update and test thoroughly or leave the same, since it's not causing any breakage and is passing CI.